packages/amplication-data-service-generator/src/server/static/src/auth/ITokenService.ts

```diff
@@ -1,5 +1,5 @@
 export interface ITokenPayload {
-  id: string;
+  id: string | number;
```

Comment:
This should be generated dynamically by the type of the ID field on the User entity (either a string or a number)